### ISSUE TITLE
NormalizeWhitespace do not add space after tuple type in parameter and in return

### ISSUE THREAD

**Issue by werwolfby** [dotnet/roslyn#29390]
**Version Used**: 
The same behavior on *2.8.2*, *2.9.0*

**Steps to Reproduce**:

1. Create method declaration:
```csharp
static void Main(string[] args)
{
    var methodDeclaration = SyntaxFactory.MethodDeclaration(
        new SyntaxList<AttributeListSyntax>(),
        SyntaxFactory.TokenList(SyntaxFactory.Token(SyntaxKind.PublicKeyword)),
        SyntaxFactory.ParseTypeName("(string prefix, string uri)"),
        null,
        SyntaxFactory.Identifier("XmlPoke"),
        null,
        SyntaxFactory.ParameterList(
            SyntaxFactory.SeparatedList(new[]
            {
                SyntaxFactory.Parameter(
                    new SyntaxList<AttributeListSyntax>(),
                    SyntaxFactory.TokenList(),
                    SyntaxFactory.ParseTypeName("(string prefix, string uri)"),
                    SyntaxFactory.Identifier("ns"),
                    null)
            })),
        SyntaxFactory.List(new List<TypeParameterConstraintClauseSyntax>()),
        null,
        null
    );

    Console.WriteLine(methodDeclaration);
    Console.WriteLine(methodDeclaration.NormalizeWhitespace());
}
```

2. Run it

**Expected Behavior**:
Add whitespace between name and tuple type in return statement and in parameter definition:
```csharp
public (string prefix, string uri) XmlPoke((string prefix, string uri) ns)
```

**Actual Behavior**:
There are missed whitespaces (github even can't highlight this code correctly).
```csharp
public (string prefix, string uri)XmlPoke((string prefix, string uri)ns)
```

I don't think that anyone expect such behavior.
If we replace `tuple` with array of tuple that everything will be correct:

```csharp
public (string prefix, string uri)[] XmlPoke((string prefix, string uri)[] ns)
```

**Comment by werwolfby** [comment]
I'm going to fix it.

**Comment by jcouv** [comment]
Merged for 16.0 preview 2.
